.github/workflows/run_periodic_tests.yml

```diff
@@ -111,6 +111,10 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: nox -s doctests
 
+      - name: Check if the documentation can be built
+        if: matrix.os == 'ubuntu-latest'
+        run: nox -s docs
+
```

Comment:
Because I noticed some errors and warnings are not caught by the doctests, this can help ensure we are free of them. We could move the examples job above in the code too so that it can run in case the docs and/or the doctests are going to fail.